src/domain/metadata/builders/MetadataPayloadBuilder.ts

```diff
@@ -131,6 +142,9 @@ export class MetadataPayloadBuilder {
             ...(categoryOptionCombos && {
                 categoryOptionCombos: this.excludeDefaultMetadataObjects(categoryOptionCombos, removeCategoryObjects),
             }),
+            ...(visualizations && {
+                visualizations: visualizationsWithRows,
```

Comment:
it's conceptually more fitting that the checked var and the value is the same, so `visualizationsWithRows`. So we set to `undefined`, not `[]` above in the code for the no-value.